app/models/vedastore/ballot_style.rb

```diff
@@ -0,0 +1,13 @@
+class Vedastore::BallotStyle < ActiveRecord::Base
+  include XsdRailsFunctions
+  include Vedaspace::BallotStyle
+  
+  belongs_to :election
+  has_one :external_identifier_collection, :as=>:identifiable
+  has_many :ordered_contests
```

Comment:
I see, that makes sense. I think my instinct is to derive all the VEDaStore storage relationships, like you're currently doing with belongs_to, in the interest of isolating the relationship definitions to VEDaSpace. There's probably some subtleties there I'm not thinking about though.

In any event, if the VEDaStore models are written and working, I suspect it's not a huge priority.